src/_repobee/cli/dispatch.py

```diff
@@ -225,7 +221,39 @@ def _raise_illegal_action_error(args: argparse.Namespace) -> NoReturn:
     )
 
 
-def _handle_hook_results(hook_results, filepath):
+def _handle_hook_results(
+    args: argparse.Namespace, hook_results: Mapping
+) -> None:
+    if _should_echo_hook_results(args, hook_results):
```

Comment:
Just a note here - `should_print_hook_results_to_STDOUT` is a better name because `echo` is specific to mainly Unix-style operating systems. However, repobee doesn't support Windows yet so it's cool.

Reply:
It refers to the function in the plugin system that's called, `plug.echo`. It prints to stdout and to the log file. So it's a bit more specific than printing to stdout.